src/fun/counting_v2.go.tmpl

```diff
@@ -0,0 +1,271 @@
+		{{$userCount:=dbGet $user.ID "counting"}}
+		{{if not $userCount}}
+			{{sendMessage nil (cembed 
+				"title" (print "No available stats") 
+				"description" (printf "`%s` has yet to count ☹️\nMaybe give them a heads-up to come join?" 
+					$user.Username
+				) 
+				"footer" (sdict "text" "Use this command: -CStats [User: @/ID]") 
+				"color" 16711680
+			)}}
+		{{else}}
+			{{$userCount:=toInt $userCount.Value}}
+			{{$userCorrect:=(dbGet $user.ID "countingCorrect").Value}}
+			{{sendMessage nil (cembed 
+				"title" (printf "🔢 %s's Counting Statistics" $user.Username) 
+				"description" (printf "%s has counted **%d times**\n**%d** have been correct\nThis makes their average **%.2f%%**" 
+					$user.Mention $userCount (toInt $userCorrect) (div $userCorrect $userCount|mult 100.0)
+				) 
+				"footer" (sdict "text" "Use this command: -CStats [User: @/ID]") 
+				"color" 30654
+			)}}
+		{{end}}
+
+	{{else if eq (index $re2 0 2|lower) "leaderb" "lb"}} {{/* leaderboard */}}
+		{{$desc:=""}}{{$pos:=1}}
+		{{range dbTopEntries "countingCorrect" $leaderboardLength 0}}
+			{{- $desc =printf "%s\n#%-3d %4d - %-4s" $desc $pos (toInt .Value) (or (userArg .UserID) (str .UserID))}}
+			{{- $pos =add $pos 1 -}}
+		{{end}}
+		{{sendMessage nil (cembed 
+			"author" (sdict 
+				"icon_url" (.Guild.IconURL "512") 
+				"name" "Counting Leaderboard"
+			) 
+			"description" (printf "```Pos    ✅  User%s```" $desc) 
+			"footer" (sdict "text" "Use this command: -CStats Leaderboard") 
+			"color" 30654
+		)}}
+
+	{{else}}
+		{{sendMessage nil (cembed 
+			"title" "Invalid Syntax" 
+			"description" "For server counting statistics: `-CStats`\nFor a members' statistics: `-CStats <User: @/ID>`\nFor server leaderboard: `-CStats Leaderboard`" 
+			"color" 16744192
+		)}}
+	{{end}}
+	{{return}}
+{{end}}
+
+{{if and (eq $db.last.user .User.ID) (not $countTwice)}}
+	{{sendMessage nil (cembed 
+		"description" (printf "You can't count twice in a row 🥲\nThe next number is %d" $db.next) 
+		"color" 16744192
+	)}}
+	{{return}}
+{{end}}
+
+{{$invalid:=false}}
+{{$re:=reFindAll `\b([%e]|abs|pi)(\b|$)|[+\-*/^π()]|a?sin|a?cos|a?tan|a?cot|a?sec|a?csc|sqrt|l(o?g|n)|\d+|[MDCLXVI]M{0,3}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|V?I{0,3})` .Cmd}}`{{json $re}}`
+{{if eq (joinStr "" $re) ""}}
+	{{return}}
+{{end}}
+{{if ne .Cmd (joinStr "" $re)}}
+	{{$invalid =true}}
+{{end}}
+
+{{$convertRN:=sdict "I" 1 "V" 5 "X" 10 "L" 50 "C" 100 "D" 500 "M" 1000}}
+{{$exp:=""}}{{$lastNumeral:=false}}
+{{range $re}}
+	{{- $roman:=reFindAll `I|V|X|L|C|D|M` .}}
+	{{- if eq (len $roman) 0}}
+		{{- $exp =printf "%s%s" $exp .}}
+		{{- if reFind `\d+` .}}
+			{{- if $lastNumeral}}
+				{{$invalid =true}}{{- break}}
+			{{- end}}
+			{{- $lastNumeral =true}}
+		{{- else}}
+			{{- $lastNumeral =false}}
+		{{- end}}
+		{{- continue}}
+	{{- end}}
+	{{- if $lastNumeral}}
+		{{$invalid =true}}{{- break}}
+	{{- end}}
+	{{- $values:=cslice}}{{- $value:=0}}{{- $skip:=false}}
```

Comment:
```suggestion
	{{- $values:=cslice}}{{$value:=0}}{{$skip:=false}}
```

Reply:
Not needed because they're on the same line? That would be a new idea to me

Reply:
Trim markers only trim whitespace from the action they're added to and the last/next non-whitespace character.